godtools/Managers/ToolsManager.swift

```diff
@@ -54,11 +63,13 @@ extension ToolsManager: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: ToolsManager.toolCellIdentifier) as! HomeToolTableViewCell
+        cell.titleLabel.text = self.resources![indexPath.section].name
```

Comment:
What do you think about making functions to pass the data to the cellview instead of setting the values here? it may be much cleaner I believe